merlin/models/tf/models/base.py

```diff
@@ -1692,6 +1701,7 @@ def build(self, input_shape=None):
         input_shape : tf.TensorShape, optional
             The input shape, by default None
         """
+        super().build(input_shape)
```

Comment:
calling super().build doesn't work in the general case with out input types. I'm going to update to perform a model call with data in `from_config` directly which is what this line is intended to achieve